heroku.md

```diff
@@ -0,0 +1,182 @@
+account then transfer ownership to the shared account when it is
+ready for billing. This is how DLAD works and so far it is fine.
+
+Not sure if something is too expensive to use as an add-on? Ask.
+There is no fixed rule, so getting other thoughts on the cost /
+benefit is the best current option.
+
+
+## Pipelines
+
+TL;DR: Use [pipelines](https://devcenter.heroku.com/articles/pipelines).
+They are :mega: :rainbow: :beers: :fireworks:
+
+We have staging sites tied to our Master branches. Merge code into
+master and staging builds will update automatically (after tests
+pass).
+
+Before merging, we have PR builds spin up with staging settings into
+new apps. You can access the link to the apps in the PR itself.
+These are helpful when asking non-developers to sign off on a
+feature before it is merged.
+
+PR builds copy ENV from Staging (dev note: `app.json` does the magic
+on this in project root).
+
+Staging and Production ENV are entirely separate. Changing one does
+NOT change the other. This will make you confused one day.
+
+On deploy or ENV change, the application will restart. It is
+normally a quick blip and doesn’t require downtime announcements.
+
+
+## Production Deploy with Pipelines Cheatsheet
+
+### Tag a release in GitHub with release notes that are useful
+
+The tag is purely informational. It is not currently used to push
+to production using Pipelines. Pushing to prod moves whatever is on
+staging, which is quite likely `master` unless you are doing weird
+things (note: don't do weird things).
+
+### Confirm everything is all good on staging.
+
+Staging probably has `master`. If it works, pushing to prod should
+be :rainbow: unless you forgot you'll need changes to ENV for any new
+features you may be pushing if so, make said changes!
+
+Get stakeholder sign off on the Pipeline staging app if it seems
+appropriate (i.e. only push stuff that has been approved to go live
+and / or is a bug / security fix)… but ideally that has been done
+before merge with PR builds… right? right‽
+
+### Login to Heroku Dashboard
+
+On the app Pipeline page, click "Promote to Production"
+
+Choose the "compare on GitHub option" to confirm the change set is
+indeed what you expect it to be. If it is, go back and click
+"Promote". If it is not, determine why it is not what you expect then
+make decide what to do about that.
+
+### Confirm the app restarts and seems good
+It takes a few seconds to restart. If things are _not_ good, rollback
+in the Dashboard.
+
+### Inform stakeholders if appropriate
+
+You just made a change to production. Tell people that care.
+
+
+## Manual Deploys
+
+You can still of course manually deploy when using Pipelines,
+but it is best to do this to staging and then promote from there
+rather than just hoping everything is going to be great in
+production.
+
+
+## Viewing / changing Environment Variables
+
+Using ENV for config is common for Heroku deploys. You can view and
```

Comment:
Do/should we have a policy about keeping sensitive ENV variables in the shared notes folder in lastpass?